Makefile

```diff
@@ -95,9 +95,13 @@ help-extended: #HELP Display extended help.
 #SECTION Development
 
 .PHONY: lint
-lint: $(GOLANGCI_LINT) #HELP Run golangci linter.
+lint: lint-custom $(GOLANGCI_LINT) #HELP Run golangci linter.
 	$(GOLANGCI_LINT) run --build-tags $(GO_BUILD_TAGS) $(GOLANGCI_LINT_ARGS)
 
+.PHONY: lint-custom
+lint-custom: custom-linter-build
+	go vet -vettool=./bin/custom-linter ./...
```

Comment:
`golangci-lint` should already be running `go vet` (assuming the `govet` linter is enabled). So I think we get this part automatically if we add the directory where we build the linter binary to the path for the `golangci-lint run` command.

Reply:
See https://golangci-lint.run/contributing/new-linters/
But I could not make it work. 
I would need to spend more time on that.

Reply:
Created : https://github.com/operator-framework/operator-controller/issues/1775